index.js

```diff
@@ -42,27 +47,35 @@ const parseEvent = (event) => {
 };
 
 // Prepare the Messages and Options
-const prepareLogs = (eventData) => {
+const prepareLogs = (eventData, raw_event_message_requested) => {
     return eventData.logEvents.map((event) => {
+        const raw_event_message = raw_event_message_requested && typeof event.message === 'string';
+        const eventMetadata = {
+            source: 'cloudwatch'
+            , event: {
+                type: eventData.messageType
+                , id: event.id
+            }, log: {
+                group: eventData.logGroup
+                , stream: eventData.logStream
+            }
+        };
+
+        const line = raw_event_message ? event.message : JSON.stringify(Object.assign({}, {
+            message: event.message
+        }, eventMetadata));
+
+        let meta = {
```

Comment:
Nit: `const` instead of `let`

Reply:
@matt-march, well, tried - can't since I override it later.

Reply:
Whoops, I even commented on that line too. 🤦

Reply:
@matt-march, restructured a little bit